kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -167,7 +167,7 @@ import ShowFor from "sumo/js/showfor";
             // Opera already seems to trigger the `click` event when Enter is pressed
             if (!($.browser.opera && event.keyCode === 13)) {
               event.preventDefault();
-              $detailsSummary.click();
+              $detailsSummary.on("click", );
```

Comment:
This should be a trigger.
```suggestion
              $detailsSummary.trigger("click");
```